docs/introduction.mdx

```diff
@@ -19,20 +19,15 @@ import Admonition from '@theme/Admonition';
 import ImageSwitcher from "@site/src/components/ImageSwitcher.js";
 
 <Admonition type="caution" icon="" title="">
-Beyond Identity will be releasing a new, consolidated admin experience that incorporates our existing Secure Workforce product with two new products, Device360 and our Next Gen SSO. Post launch, Beyond Identity will communicate a migration plan to current customers.
+Beyond Identity has released a new, consolidated admin experience that incorporates our next generation MFA and Zero Trust products with new products, Access360, Device360, and Secure SSO. Beyond Identity will communicate a migration plan to current customers.
 </Admonition>
 
-<Box sx={{ width: '100%', display: 'flex', justifyContent: 'center', paddingTop: '72px', paddingBottom: '72px' }}>
-  <Stack direction={{ xs: 'column', sm: 'row' }} spacing={3} alignItems="center" sx={{ width: '100%' }}>
-  <Box sx={{ flexGrow: 1, flexShrink: 1, flexBasis: 'auto' }}>
-    <h1>Beyond Identity Secure Access Platform Docs</h1>
-    <p>Know and control exactly who and what device is accessing your resources. Stay ahead of bad actors with all the clarity, control and precision you need in order to make informed security decisions, all in one platform.</p>
-  </Box>
-  <Box sx={{ flexShrink: 0 }}>
-    <img src="/img/docs-image.png" alt="Docs" style={{ width: '500px', height: 'auto', maxWidth: '100%', border: 'none' }} />
-  </Box>
-  </Stack>
-</Box>
+<h1>Beyond Identity Secure Access Platform Docs</h1>
+    <p>With the Secure Access Platform, you can use our phish-resistant, passwordless MFA, our lightweight, Secure SSO, and a suite of access risk detection features all in one platform, for a modern access security approach that’s easy on admins and users alike.</p>
+    <p>You can use one feature or all three features based on the package you purchase. </p>
+
+
+ ![Secure Access Platform](images/what-is-secure-access.png)
```

Comment:
I think we should leave the image as is on the homepage. The "what is secure access" image should be added on another page, maybe in one of the introduction pages.